articles/events_executor.md

```diff
@@ -129,11 +132,63 @@ This class will forward the items needed for pushing events to the notify guard
 Whenever an entity is associated to an executor it may already have multiple items of work ready to be processed.
 The `EventsExecutor` should tell the entity whether it has to immediately push or to discard those pending events.
 
+### Types of Events queue
+
+There are multiple uses cases with different requirements (performance, determinism in events ordering, bounded memory). A single `EventsQueue` can't comply with all requirements simultaneously, so there's need for different types of queue.
+
+##### SimpleEventsQueue
+
+This is the simplest queue, as it does not peform any checks when events are pushed or extracted, so it's more suitable for situacions where CPU performance is needed.
```

Comment:
I would just say that this is the simplest queue, just showing how to wrap std::queue into the api

Reply:
Done in 0d70bf2